server/lib/tuist/environment.ex

```diff
@@ -446,6 +446,10 @@ defmodule Tuist.Environment do
     get([:secret_key, :tokens], secrets, default_value: secret_key_base(secrets))
   end
 
+  def secret_key_encryption(secrets \\ secrets()) do
```

Comment:
I introduced this key, which we can use across any other field that needs encryption.